src/applications/disability-benefits/new-686/config/chapters/report-add-a-spouse/current-spouse-marriage-history-details/helpers.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export function spouseTitle({ formData }) {
```

Comment:
NIT: According to the React [docs](https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized), I think this is supposed to be capitalized.

```
- export function spouseTitle({ formData }) {
+ export function SpouseTitle({ formData }) {
```

Reply:
Refactored [here](https://github.com/department-of-veterans-affairs/vets-website/pull/11820/commits/ff8536a2d81259a55660938c1de9296259bda912)